src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -372,23 +370,62 @@ def minimize_corpus(self, target_path, arguments, input_dirs, output_dir,
     runner = libfuzzer.get_runner(target_path)
     libfuzzer.set_sanitizer_options(target_path)
     merge_tmp_dir = self._create_temp_corpus_dir('merge-workdir')
-
-    merge_result = runner.merge(
-        [output_dir] + input_dirs,
-        merge_timeout=max_time,
-        tmp_dir=merge_tmp_dir,
-        additional_args=arguments,
-        artifact_prefix=reproducers_dir)
-
-    if merge_result.timed_out:
-      raise MergeError('Merging new testcases timed out')
-
-    if merge_result.return_code != 0:
-      raise MergeError('Merging new testcases failed')
+    merge_control_file = os.path.join(merge_tmp_dir, 'MCF')
+
+    # Make sure the merge control file does not exist at this point.
+    shell.remove_file(merge_control_file)
+
+    additional_args = arguments + [
+        '%s%s' % (constants.MERGE_CONTROL_FILE_ARGUMENT, merge_control_file)
```

Comment:
one more comment: we should just use '+' in such cases. 

See http://google.github.io/styleguide/pyguide.html#310-strings (first example of "No").

Reply:
Oh, nice, done!